cheatsheet.md

```diff
@@ -11,98 +11,103 @@ with possibly out-of-date explainers.
-
-## Activation algorithms
-
-This is a list of algorithms that cause the element to activate. When activated,
-a `rendersubtreeactivation` signal is fired on the element and the `rendersubtree`
-property is set to `""`. Note that `skip-activation` implies
-`skip-viewport-activation` when disabling a particular algorithm.
-Note also that non-activatibility applies to every descendant of a locked element.
-An element within a non-activatable locked subtree is not activatable, even if it has
-other locked ancestors that are activatable.
+* **Locked element**: the element's subtree is not being rendered or hit-tested.
+* **Unlocked element**: the element's subtree is being rendered and hit-tested.
+  Note that this state is independent of whether render-subtree: invisible
+  property applies to the element. See below for more information.
+* **render-subtree**: this is an alternative feature name. It refers to the CSS
+  property `render-subtree` which can take several values in order to lock an
+  element.
+  * `render-subtree: invisible`: this refers to the fact that this
+    element's locked state is being managed by the user-agent.  Specifically,
+    the user-agent will unlock elements as they approach the viewport and lock
+    them as they move away from the viewport. Locked content is discoverable by
+    user-agent algorithms which may bring the element into view, causing it to
+    be unlocked. This configuration is in the initial proposal.
+  * `render-subtree: invisible skip-activation`: this refers to the
+    fact that the subtree is locked and the user-agent will not manage the
+    state. The content is not discoverable by any user-agent algorithms.
+    This configuration is in the initial proposal.
+  * `render-subtree: invisible skip-viewport-activation`: this refers to the
+    fact that the subtree is locked and the user-agent will not manage the
+    state. The content is, however, discoverable by user-agent algorithms, which
+    may fire events with targets in the locked subtree.
+    This configuration is tentative and is not a part of the initial proposal.
+    Specifically, the details of the event have not been discussed in detail
+    yet.
+  * Note that all of the above configurations enforce `contain: style layout;`.
+    Additionally, if the element is locked it also enforces `contain: size;`.
+    This containment is additive and takes effect on top of any other
+    containment specified.
+* **contain-intrinsic-size**: this refers to a complementary CSS feature called
+  contain-intrinsic-size, which allows the developer to specify a placeholder size
+  when size containment is present, as it would be when the subtree is locked.
+
+## User-agent algorithms
+
+This is a list of algorithms that can bring an element into view or otherwise
+cause the user-agent to unlock the elements. Note that unless explicitly stated
+otherwise, the language below uses 'unlock' as the action since that is the
```

Comment:
Can you use another term instead? It is a bit confusing :D

Reply:
Do you have any suggestions? I kind of want to avoid the word 'activate' since it's been used to mean unlock in many of our discussions so it would be confusing. I think this should be the same word as this signal that we want to add. However, I also don't know of a good name for it.

Reply:
Maybe we can just keep "activate" but explain here? Or maybe, "unlock or send activation signal (depending on the render-subtree value)"?

Reply:
I'm going to commit this as is since there's a bunch of other updates that should go in. We can follow-up in an issue about what to call this action.